fastlane/lib/fastlane/actions/make_changelog_from_jenkins.rb

```diff
@@ -37,7 +37,12 @@ def self.available_options
           FastlaneCore::ConfigItem.new(key: :fallback_changelog,
                                        description: "Fallback changelog if there is not one on Jenkins, or it couldn't be read",
                                        optional: true,
-                                       default_value: "")
+                                       default_value: ""),
+          FastlaneCore::ConfigItem.new(key: :include_commit_body,
+                                       description: "Include the commit body along with the summary",
+                                       optional: true,
+                                       is_string: false,
+                                       default_value: false)
```

Comment:
does this `default_value` false means the current default behaviour is changed ? That's what your meant ?